src/thunderbird_accounts/infra/views.py

```diff
@@ -0,0 +1,7 @@
+from django.http import HttpRequest, HttpResponse
+
+
+def health_check(request: HttpRequest):
+    """Do nothing, only prove the service is responsive."""
+
+    return HttpResponse('Aliveness demonstrated!')
```

Comment:
On appointment we demonstrate connectivity and aliveness. Did we want to do a db query here?

Reply:
(Although I think the middleware technically does touch the db, so I think we may be fine.)